File: URI-1013_O_Maior.py
/*-------------------*
| Rodrigo CavanhaMan |
| URI 1013           |
| O Maior            |
*--------------------*/
A,B,C = map(int,input().split() )
MaiorAB = ((A+B+abs(A-B))/2)
MaiorABC = ((MaiorAB+C+abs(MaiorAB-C))/2)
print("%.0i eh o maior" %MaiorABC)

File: CPP/URI-1013_O_Maior.cpp
/*-------------------*
| Rodrigo CavanhaMan |
| BEE 1013           |
| O Maior            |
*--------------------*/
#include<stdio.h>
#include <stdlib.h>     /* abs */
using namespace std;

int main() {

	int a,b,c,maiorAB, maiorABC;
	scanf("%d",&a);
	scanf("%d",&b);
	scanf("%d",&c);

	maiorAB = (a+b + abs(a-b))/2;
	maiorABC = (maiorAB+c + abs(maiorAB-c))/2;

	printf("%d eh o maior\n",maiorABC);

	return 0;
}
